src/utils.h

```diff
@@ -92,6 +92,7 @@ namespace Utils {
 	 */
 	std::string EncodeUTF(const std::u32string& str);
 
+#ifndef __amigaos4__
```

Comment:
no wstring support?

Reply:
Not yet, and I didn't feel like hacking around, because these functions are only used on Windows.